eessi/reframe/config/config_hortense_gpu_rome_a100.py

```diff
@@ -0,0 +1,158 @@
+from reframe.core.backends import register_launcher
+from reframe.core.launchers import JobLauncher
+
+account = "my-slurm-account"
+
+# use 'info' to log to syslog
+syslog_level = 'warning'
+
+perf_logging_format = 'reframe: ' + '|'.join([
+    'username=%(osuser)s',
+    'version=%(version)s',
+    'name=%(check_name)s',
+    'system=%(check_system)s',
+    'partition=%(check_partition)s',
+    'environ=%(check_environ)s',
+    'num_tasks=%(check_num_tasks)s',
+    'num_cpus_per_task=%(check_num_cpus_per_task)s',
+    'num_tasks_per_node=%(check_num_tasks_per_node)s',
+    'modules=%(check_modules)s',
+    'jobid=%(check_jobid)s',
+    '%(check_perfvalues)s',
+])
+
+format_perfvars = '|'.join([
+    'perf_var=%(check_perf_var)s',
+    'perf_value=%(check_perf_value)s',
+    'unit=%(check_perf_unit)s',
+]) + '|'
+
+
+@register_launcher('mympirun')
+class MyMpirunLauncher(JobLauncher):
+    def command(self, job):
+        return ['mympirun', '--hybrid', str(job.num_tasks)]
+
+
+site_configuration = {
+    'systems': [
+        {
+            'name': 'hortense',
+            'descr': 'Hortense',
+            'hostnames': ['login.*.dodrio.os'],
+            'modules_system': 'lmod',
+            'partitions': [
+                {
+                    'name': 'gpu_rome_a100',
+                    'scheduler': 'slurm',
+                    'access': [f'-A {account} --export=NONE --get-user-env=60L --partition=gpu_rome_a100'],
```

Comment:
The `gpu_rome_a100` partition now has both 40GB and 80GB A100 GPUs, so it would be better to switch to the specific `gpu_rome_a100_40` and `gpu_rome_a100_80` partitions.

And while we're at it, why not also add the CPU-only partitions.

You can easily instruct ReFrame to only submit tests to a specific partition using an option like `--system hortense:gpu_rome_a100_40gb`.